### DISCUSSION TITLE
Is it possible to validate a single Controller field on different mode than the rest of the form?

### DISCUSSION THREAD

**Discussion by nbnsresta** [react-hook-form/react-hook-form#4183] [Q&A] [answered]
Prelude: I have been using v5.7.2 still. 

How could it be possible for a single Controller field to run static/dynamic validation on mode onChange/onBlur while the rest of the form validates on default validation mode.

The reason that this is needed is because the validation is async and still should be part of the validation pipeline while being triggered on change/blur events.

**Comment by bluebill1049** [comment, marked as answer]
You probably will have to use the `trigger` method.
